Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [workspace]
-
+resolver = "2"
```

Comment:
I think this is not needed here, because of [this](https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html) update.